tutorindigo/templates/indigo/env.config.jsx

```diff
@@ -1,8 +1,24 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import Cookies from 'universal-cookie';
 
 import Footer from '@edly-io/indigo-frontend-component-footer';
+import { getConfig } from '@edx/frontend-platform';
 import { DIRECT_PLUGIN, PLUGIN_OPERATIONS } from '@openedx/frontend-plugin-framework';
 
+
+const AddDarkTheme = () => {
+  const cookies = new Cookies();
+  const themeCookieName = getConfig().THEME_COOKIE_NAME;
```

Comment:
Hardcode "indigo-toggle-dark". That cookie does not have any value. It's either absent or present (with empty value).